docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -336,51 +338,38 @@ Use the following steps to manually trigger a repave on a hybrid node pool.
 
    ![Edit Hybrid Profile](/aws_eks-hybrid_create-hybrid-node-pools_in-use-clusters.webp)
 
-5. Click on the **edge-nodeadm x.y.z** Kubernetes layer to view the **Edit Pack** page.
+5. Click on the version drop-down and select **Create new version**. The version creation dialog appears.
 
-6. In the YAML editor on the right, add an arbitrary node label to `cluster.config.kubelet.flags`.
+6. Fill in **1.1.0** in the **Version** input field and click **Confirm**. The new cluster profile version is created
```

Comment:
Do we want to be prescriptive here? Telling them to just increment the version is a bit more accurate / future proof.